src/SymbolServer.jl

```diff
@@ -27,6 +27,17 @@ mutable struct SymbolServerInstance
     end
 end
 
+function remove_non_general_pkgs!(pkgs)
+    reg_pkgs = Pkg.Types.Context().registries[3].pkgs
```

Comment:
I think this needs to be changed to find wich element in `registries` is actually the general registry, right? On my system for example the length of it is just 1, so I assume you have two additional registries on your system?

Reply:
Whoops, yes, I meant to leave a comment asking how you'd typically get General in the extension